Vagrantfile.pkg

```diff
@@ -13,7 +13,15 @@ _config = {
         "/var/www" => File.join(Dir.pwd, "www"),
```

Comment:
@stevenrombauts I know about the vagrantfile, but what does the vagrantfile.pkg do?

Reply:
@yiendos The `.pkg` file is the Vagrantfile that we package with the published box. So the actual `Vagrantfile` is only used when you checkout the repository and run it yourself. The `Vagrantfile.pkg` is what Vagrant will use if you download the box (using `vagrant init joomlatools/box`

Reply:
It's used in the Packer build instructions here: https://github.com/joomlatools/joomlatools-vagrant/blob/master/packer.json#L102-L107